docs/appendices/release-notes/5.9.0.rst

```diff
@@ -47,6 +47,9 @@ Breaking Changes
 
 - Removed deprecated ``validation`` parameter from ``COPY FROM``.
 
+- Added validation of unknown or irrelevant for the ``file`` scheme properties
+  for the ``COPY FROM`` and ``COPY TO`` statements.
```

Comment:
We actually already have validation of values (for example compression accepts only gzip), that's why I elaborated on what kind of validation is done. 

Given that you said it's not really a breaking change
>I am not sure if the third one could be seen a a breaking change to some users

maybe I should drop this change entry completely?

Reply:
> maybe I should drop this change entry completely?

I think we could have more inputs on this. I personally cannot decide which would be better, silently ignoring vs throwing exceptions.

Reply:
Is it fine if I merge it with the change entry (because i need this for https://github.com/crate/crate/pull/16468) and  later we can drop the change entry (if needed)

Reply:
@mfussenegger do you think we need such validation (for example, eject S3 settings when using Azure scheme)?

If yes, it looks like it's a breaking change? See `testCopyFromPlanWithParameters`

Also, if we say we need such validation, for non-file schemes we can do it only in plugins, because only plugins are aware about scheme specific settings. See https://github.com/crate/crate/pull/16468#issuecomment-2272988294